src/tests/system/tests/test_passkey.py

```diff
@@ -26,6 +26,26 @@ def passkey_requires_root(client: Client) -> tuple[bool, str] | bool:
     return True
 
 
+def umockdev_ipaotpd_update(ipa: IPA) -> bool:
```

Comment:
```
@pytest.fixture
def umockdev_ipaotpd_update(mh):
    ipa: IPA = mh.fixtures["ipa"]
    ...
```

Is this required for all IPA tests or for this new one?

Reply:
Not for add but for this one and whatever will add where I use the `user_auth_type=`.

I tried, but it does not work when I add with `scope`. I tried all the scope.


```
INFO     pytest_mh.logger:logging.py:162 COLLECT ARTIFACTS DONE :: ipa
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> traceback >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
ScopeMismatch: You tried to access the function scoped fixture mh with a session scoped request object, involved factories:
tests/test_passkey.py:30:  def umockdev_ipaotpd_update(mh)
../../../../../.local/lib/python3.11/site-packages/pytest_mh/_private/fixtures.py:355:  def mh(request: 'pytest.FixtureRequest') -> 'Generator[MultihostFixture, None, None]'
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> entering PDB >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>

>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> PDB post_mortem >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
> /usr/lib/python3.11/site-packages/_pytest/fixtures.py(698)_check_scope()
-> fail(
(Pdb) c


```

Reply:
Ok, "function" should work and works locally for me. But it might be better to just place the fixture code inside the test for the moment.

I realized that even we can workaround things by accessing `mh` directly, the setup functions were not yet run so this use case is unsafe at this moment.

Reply:
Actually scratch previous comment, if `mh` fixture is available, it is already fully setup.

Reply:
By the way, I opened https://github.com/next-actions/pytest-mh/pull/57 so we are able to use the host roles directly.

Reply:
updated